.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: Docs - Build and Deploy
+
+on: [push]
+
+jobs:
+  build:
+    name: Build docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with: 
+          python-version: '3.8'
+
+      - name: Install dependencies
+        run:
+          python -m pip install --upgrade pip
```

Comment:
why is this step necessary?

Reply:
It's not strictly necessary but I find that's ussualy a good practice to update to the latest pip version before using it on isolated environments.